packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -402,10 +448,14 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
                 handleQueryListKeyUp?.(e);
             }
 
-            this.props.popoverTargetProps?.onKeyDown?.(e);
+            this.props.popoverTargetProps?.onKeyUp?.(e);
```

Comment:
yikes - looks like this was always incorrect? let's be very careful with changes to this component because it seems it's current focus interactions may not be extensively tested - could we remove this if not required for this PR and track with an issue?

Reply:
yeah sounds good- i was concerned this might break existing workflows in the super weird case someone relied on this behavior

Reply:
yea, I imagine we can change it I just don't want to think through it now alongside this bigger feature change. thank you!